### DISCUSSION TITLE
请问paddle版本的几个模型在哪下载呀

### DISCUSSION THREAD

**Discussion by jweden33** [RapidAI/TableStructureRec#98] [Q&A]
请问paddle版本的几个模型在哪下载呀 obj_det_paddle，edge_det_paddle，cls_det_paddle

**Comment by Joker1212** [comment]
https://www.modelscope.cn/models/jockerK/TableExtractor

**Comment by jweden33** [comment]
![image](https://github.com/u
![857](https://github.com/user-attachments/assets/d4c1dcab-9027-48a9-b67c-ea69aedeadf8)
ser-attachments/assets/315c12cf-b77f-4349-a057-3eed22e3d8a5)
好像不是这几个吧

**Reply by Joker1212** [reply]
是指的表格检测吧，就是在这里了，点开文件夹，rapid_table_paddle,因为表格检测有paddle有Yolo的模型，也有转换得到的onnx

**Reply by jweden33** [reply]
抱歉 我傻了 没点进文件夹去看
